tests/spec/core/algorithms-spec.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+describe("Core — Algorithm Lists", () => {
+  afterAll(flushIframes);
+  it("should add assert class to Assertions in algorithms", async () => {
+    const ops = {
+      config: makeBasicConfig(),
+      body:
+        makeDefaultBody() +
+        `<section>
+            <ol class='algorithm'>
+            <li>This one shouldn't be modified.</li>
+            <li>Assert: This one should have assert class added to it.</li>
```

Comment:
This is looking good, but let's add a few more test cases... what happens if you have:
```HTML
  <li><!-- whitespace -->
     Assert: test
  <li>
```